store/root/store.go

```diff
@@ -77,10 +78,13 @@ func New(
 	}
 
 	pruningManager := pruning.NewManager(logger, ss, sc)
+	pruningManager.SetStorageOptions(ssOpts)
+	pruningManager.SetCommitmentOptions(scOpts)
+	pruningManager.Start()
 
 	return &Store{
 		logger:          logger.With("module", "root_store"),
-		initialVersion:  initVersion,
```

Comment:
i dont think this is correct, height 1 is not always guranteed

Reply:
This is the default height. If a great height is indicated via `InitChain`, `SetInitialVersion` is called.